src/assets/githubAction/runPlaywrightOnCI.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Playwright and GitHub Actions - Run tests in CI
+authorName: Nabin Ale
+authorAvatar : https://avatars.githubusercontent.com/u/61624650?v=4
+authorLink: https://github.com/nabim777
+createdAt: Apr 8, 2025
+tags: continuous integration, continuous delivery, continuous deployment, github actions, ci, cd, playwright
+banner: https://blog.jankaritech.com/src/assets/RunPlaywrightOnCI/images/githubaction_with_playwright_banner
+---
+
+This is a blog about how to run Playwright UI tests in GitHub Actions. If you are not familiar and new to UI testing using Playwright, then it would be good to check out this blog about [Playwright](https://blog.jankaritech.com/#/blog/E2E%20Testing%20using%20BDD%20with%20Playwright/Behavior%20Driven%20Development%20(BDD)%20using%20Playwright). Similarly, if you are unfamiliar with GitHub Actions, you can check another blog about [Introduction to GitHub Actions](https://blog.jankaritech.com/#/blog/Introduction%20to%20GitHub%20Actions%20-%20CI%20%26%20CD).
+
+## 🤔 Why to run tests on CI?
+Tests are run on CI(Continuous Integration) to ensure the code works properly in a clean and an isolated environment every time a change is made. Here are some reasons:
+
+**1. Early bug detection:**
+By automatically running tests after every code commit, you can quickly identify issues as they arise, preventing them from accumulating and causing larger problems later on.
+
+**2. Fast feedback loop:**
+Developers get immediate notification for failing tests. That way they can fix bugs immediately and iterate quickly.
+
+**3. Consistent testing environment:**
+The CI servers run tests in an uniform environment. Therefore, there is no correlation between the developer configurations.
+
+**4. Improved code quality:**
+Regular running tests in CI ensures existing functionality does not become unusable if changes are made.
+
+**5. Reduced integration issues:**
+By regularly integrating and testing the code in CI, you reduce the risk of conflicts when merging large patches of code.
```

Comment:
how does running the tests in CI help with merge conflicts?
Bob writes some code and adjusts tests => CI is green for him
Alice changes some code in a similar area and adjusts tests => CI is green for her
whoever merges first wins and the second one has to fix the merge-conflict. How does CI help with that?